stable/cluster-overprovisioner/templates/priorityclass-default.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: scheduling.k8s.io/{{ template "PriorityClass.apiVersion" . }}
+kind: PriorityClass
+metadata:
+  name: {{ .Values.priorityClassDefault.name }}
```

Comment:
Could you apply the same naming logic for PriorityClasses as we suggest for service accounts using helper templates?

See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md#using-rbac-resources

Reply:
Nevermind. I guess this doesn't make sense here.